module-check/src/test/clojure/clojure/core/typed/test/set_new.clj

```diff
@@ -0,0 +1,41 @@
+(ns clojure.core.typed.test.set_new
```

Comment:
The namespace should be called set-new, not set_new. The file name is correct though.